main.go

```diff
@@ -97,6 +115,9 @@ func pointStream(csc <-chan []simpartsim.Coords) (etherdream.PointStream, error)
 		for cs := range csc {
 			dumpInPointStream(w, cs)
 		}
+		for {
```

Comment:
Does your CPU max out at the forever loop?

I suggest controlling looping from the call to the play method as an arg. This way, a finite amount of instructions can be played any number of times versus an infinite amount of instructions requiring the killing of an application to cease.  Also, the play method can return a cancellation channel to further improve control flow.